ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/StateManager.java

```diff
@@ -748,11 +757,11 @@ protected void unpackHeader (InputObjectState os, Header hdr)
                 throw new IOException(tsLogger.i18NLogger.get_StateManager_15());
             }
             
-            long birthDate = os.unpackLong();
+            creationTimeMillis = os.unpackLong();
             
             if (tsLogger.logger.isTraceEnabled()) {
-                tsLogger.logger.trace("StateManager.unpackHeader for object-id " + get_uid()
-                        + " birth-date " + birthDate);
+                tsLogger.logger.tracef("StateManager.unpackHeader for object-id %s creationTime %s",
+                        get_uid(), Instant.ofEpochMilli(creationTimeMillis));
```

Comment:
I'm not convinced that changing the trace message from printing a long to printing a date is related to this change, normally we'd require justification for adding extra changes beyond just the fix. But since this is "just" a trace message and is unlikely to impact users I'll still approve. @jmfinelli are you still ok to approve with these  extra changes to the PR?

Reply:
WDYT if we include both of them?
```suggestion
tsLogger.logger.tracef("StateManager.unpackHeader for object-id %s creationTime %s (creationTime in milliseconds: %s)",
                        get_uid(), Instant.ofEpochMilli(creationTimeMillis), creationTimeMillis);
```

Reply:
I see no value in outputting the long value. I think most people will ask new Date(...) to resolve what the log is telling. :-)

Reply:
long values can be easily compared using scripts but a formatted date string cannot.
I don't see any benefit in producing a verbose date (which is kind of already available since log messages are timestamped). Besides, the change to the log message is unrelated to the issue at hand, if you do need to see a different message then maybe add it in an new PR.

Reply:
Okay, I reverted any log format specific change